publish-to-maven-central/SDK4Mvn.aggr

```diff
@@ -2,10 +2,10 @@
 <aggregator:Aggregation xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:aggregator="http://www.eclipse.org/cbi/p2repo/2011/aggregator/1.1.0" label="SDK4Mvn" packedStrategy="UNPACK_AS_SIBLING" type="R" mavenResult="true" versionFormat="MavenRelease" includeSources="true">
   <validationSets label="main">
     <contributions label="sdk">
-      <repositories enabled="false" location="/home/data/httpd/download.eclipse.org/eclipse/updates/4.27-I-build"/>
+      <repositories enabled="false" location="/home/data/httpd/download.eclipse.org/eclipse/updates/4.27-I202303111800"/>
```

Comment:
Can you please explain what you are doing and why you are doing it.  This repository does not exist:

https://download.eclipse.org/eclipse/updates/4.27-I20230311180

We don't currently know the correct repository so it cannot be changed right now.  Which is why I ask, what are you trying to do and why?